lib/redmine/scm/adapters/git_adapter.rb

```diff
@@ -296,10 +294,10 @@ def diff(path, identifier_from, identifier_to = nil)
         def annotate(path, identifier = nil)
           identifier = 'HEAD' if identifier.blank?
           cmd_args = %w|blame|
-          cmd_args << '-p' << identifier << '--' <<  scm_encode(@path_encoding, 'UTF-8', path)
+          cmd_args << '-p' << identifier << '--' << scm_encode(@path_encoding, 'UTF-8', path)
           blame = Annotate.new
           content = nil
-          scm_cmd(*cmd_args) { |io| io.binmode; content = io.read }
+          scm_cmd(*cmd_args) do |io| io.binmode; content = io.read end
```

Comment:
Prefer {...} over do...end for single-line blocks.<br>Do not use semicolons to terminate expressions.